index.js

```diff
@@ -108,6 +108,17 @@ function outputChangelog (argv, cb) {
   })
 }
 
+function handleExecError (err, stderr) {
+  // If exec returns an error or content in stderr, log it and exit with return code 1
+  var errMessage = stderr || (err && err.message)
+  if (errMessage) {
+    console.error(chalk.red(errMessage))
```

Comment:
i think when `err` is null, better to use `console.warn` to output the error message. Red message means real error.
Since console.error/warn's output will have its own color, should it continue to use `chalk` to control the color?